### PR TITLE
SQL-919: Add formula to concatenate strings when using a + operator

### PR DESCRIPTION
**Pull request by nbagnard**

### CODE REVIEW

File: connector/dialect.tdd [hunk 1]
```diff
@@ -638,6 +638,11 @@
       <argument type='int' />
       <argument type='real' />
     </function>
+    <function group='operator' name='+' return-type='str'>
+      <formula>($1 || $2)</formula>
```
Comment (bucaojit) [inline connector/dialect.tdd]:
```suggestion
      <formula>(%1 || %2)</formula>
```
I ran this through TDVT and was getting an `invalid token` error, looks like we want the `%` to indicate the substitution.

**Review by bucaojit — CHANGES_REQUESTED** [review]

File: connector/dialect.tdd (hunk 1 above)
Comment (nbagnard) [reply connector/dialect.tdd]: Yes, you are right! Not sure how I got the '$' in here...
Thank you for letting me know!

**Review by bucaojit — APPROVED** [review]
LGTM

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**